blobmodel/blobs.py

```diff
@@ -23,6 +24,7 @@ def __init__(
         t_drain: Union[float, NDArray],
         prop_shape_parameters: dict = None,
         perp_shape_parameters: dict = None,
+        blob_alignment: bool = True,
```

Comment:
If we want to keep the blob alignment it would probably make sense to access it in the model class. What do you think?

Reply:
I had it there first. I changed it to the forcing class to minimize interface changes since I guess most people will not care whether their blobs are aligned or not. I can add it there, it is just to send it straight to the forcing constructor.